typo (#2141)
javascript-tutorial/en.javascript.info: 2-ui/4-forms-controls/1-form-elements/article.md

</script>
```

> Unlike most other controls, `<select>` allows to select multiple options at once if it has `multiple` attribute. That feature is rarely used. In that case we need to use the first way: add/remove the `selected` property from `<option>` subelements.

Reviewer: ```suggestion
Unlike most other controls, `<select>` allows to select multiple options at once if it has the attribute, `multiple`. In such cases, we need to use the first way - add/remove the `selected` property from `<option>` subelements.
```

1. Changed the first sentence to avoid interpreting "*multiple* attribute" as "*multiple attributes*".
2. Removed "feature is rarely used" statement, as it disrupts the flow and is unnecessary(?).

Suggestions are welcome! :-)
